block-layer/codecs/dag-pb.md

```diff
@@ -44,13 +44,13 @@ This layout can be expressed with [IPLD Schemas](../../schemas/README.md) as:
 
 ```ipldsch
 type PBLink struct {
-  Hash Bytes
-  Name optional String
-  Tsize optional Int
+  Hash optional Link
```

Comment:
https://github.com/ipfs/go-merkledag/blob/749fd8717d46b4f34c9ce08253070079c89bc56d/pb/merkledag.pb.go#L83-L88

I'm not sure on the NodeBuilder side though, I assume that either way you can end up with a "maybe Link, maybe empty"?